google-home-community.groovy

```diff
@@ -3488,6 +3533,22 @@ private deviceStateForTrait_StartStop(deviceTrait, device) {
     return deviceState
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private deviceStateForTrait_StatusReport(deviceTrait, device) {
+    def statusReport = []
+    (0..<deviceTrait.numStatusTypes).each { i ->
+        statusReport << [
+            deviceTarget: deviceTrait.statusTypes[i].deviceTargetAttribute,
+            statusCode: device.currentValue(deviceTrait.statusTypes[i].deviceTargetAttribute),
```

Comment:
This has to be one of Google's predefined [error or exception codes](https://developers.home.google.com/cloud-to-cloud/intents/errors-exceptions), which Hubitat devices are unlikely to produce natively.  It seems like some sort of translation is needed, but it would have to be user-defined since there's no canonical list of Hubitat device errors.  At that point this trait really starts becoming to complex for most users to use though...

Reply:
Fair point.  Thoughts on this trait then?  I thought I could use it for a different reason, but it didn't turn out that way.  With those code exceptions, this one might not be of any value (it's meant for a security system).

I can cancel the pull request if you'd like.

Reply:
Unless you have a particular use-case in mind, I'll go ahead and close this PR.

What were you planning to use this trait for?  Maybe there's another trait that's a better fit.